### PR TITLE
feat: added port picker vue component

### PR DESCRIPTION
**Pull request by lipskij**
Added port picker vue components

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/components/port-picker/PortsInput.vue
Comment (chmelevskij) [reply src/components/port-picker/PortsInput.vue:43]: Good call.

https://github.com/betaflight/betaflight-configurator/blob/3018ac3bce26a1288bb1ee714f4e6354c2b3637e/src/components/status-bar/ReadingStat.vue#L3 is example of how i18n is used in vue here.

**Review by chmelevskij — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
